packages/dx-react-scheduler-material-ui/src/templates/drag-drop/resize.jsx

```diff
@@ -68,17 +51,41 @@ const ResizeBase = React.memo(({
   );
 });
 
-ResizeBase.propTypes = {
-  classes: PropTypes.object.isRequired,
+Resize.propTypes = {
   position: PropTypes.oneOf([POSITION_START, POSITION_END]).isRequired,
   appointmentType: PropTypes.oneOf([HORIZONTAL_TYPE, VERTICAL_TYPE]).isRequired,
   className: PropTypes.string,
   forwardedRef: PropTypes.oneOfType([PropTypes.func, PropTypes.object]),
 };
 
-ResizeBase.defaultProps = {
+Resize.defaultProps = {
   className: undefined,
   forwardedRef: undefined,
 };
 
-export const Resize = withStyles(styles, { name: 'Resize' })(ResizeBase);
+export const StyledResizeBaze = styled(Resize)(({ theme: { spacing } }) => {
```

Comment:
Why it is exported and where it is used?

Reply:
Styled div and deleted export